linkerd/docs/client_tls.md

```diff
@@ -87,3 +87,10 @@ routers:
           certPath: /certificates/cert.pem
           keyPath: /certificates/key.pem
 ```
+
+<aside class="notice">
+When using any kind of transformer with client TLS using `io.l5d.static`, ensure that the client
+TLS's `prefix` field starts with the first transformer that is applied to a client name in the dtab.
+This will ensure that TLS is initiated between linkerd and any downstream service that matches the
+client prefix.
```

Comment:
I think this could use a little more information about how to determine the full client name including the transformer prefix.  Probably the easiest way would be click the client name to expand it in the Linkerd admin dashboard.  An example would probably be helpful.

Reply:
Good point, I could add information that points to the link to the client name. Hopefully, that is available in the admin endpoint.